builder/builder.go

```diff
@@ -25,9 +25,20 @@ var dockerImageForLang = map[string]string{
 	"javascript":     "suborbital/builder-js",
 }
 
+// BuildConfig is the configuration for a Builder.
+type BuildConfig struct {
+	JsToolchain string
+}
+
+// DefaultBuildConfig is the default build configuration.
+var DefaultBuildConfig = BuildConfig{
+	JsToolchain: "npm",
```

Comment:
Should we restrict this to a set of known good values? 

How does the user install alternative tool chains in the docker image?

Reply:
> this configuration can only be set programmatically by using subo as a library

This does mostly make things better though generally its best to fail as early as possible otherwise it might as well be javascript 🔥 

Like its going to be really annoying if I set this stuff up but only build rust for a while then go to build some javascript and this thing that has been working for weeks just barfs. 

It's not a hill I care do die on though so I'll leave it to your discretion